README.md

```
@@ -72,6 +113,22 @@ python download_air_quality.py --country IN --country-wide --max-locations 10 --
 - `--list-countries`: List all available countries
 - `--parallel`: Enable parallel mode for faster downloads (requires multiple API keys)
 
+**JARTIC Archive Download (download_jartic_archives.py):**
+- `--start, -s`: Start month (YYYY-MM)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)